QMigrate/io/data.py

```diff
@@ -15,24 +15,69 @@
 import QMigrate.util as util
 
 
-class Archive(object):
+def wa_response(convert='DIS2DIS', obspy_def=True):
     """
-    The Archive class handles the reading of archived waveform data.
+    Generate a Wood Anderson response dictionary.
 
-    It is capable of handling any regular archive structure. Some minor cleanup
-    is performed to remove waveform data from stations when there are time gaps
-    greater than the sample size. There is also the option to resample and/or
-    decimate the waveform data to achieve the user-selected unified sampling
-    rate. Keeps a record of which stations had continuous data  available for a
-    given timestep.
+    Parameters
+    ----------
+    convert : str, optional
+        Type of output to convert between; determines the number of complex
+        zeros used. Options are: 'DIS2DIS', 'VEL2VEL', 'VEL2DIS'
+    obspy_def : bool, optional
+        Use the ObsPy definition of the Wood Anderson response (Default).
+        Otherwise, use the IRIS/SAC definition.
+
+    Returns
+    -------
+    WOODANDERSON : dict
+        Poles, zeros, sensitivity and gain of the Wood-Anderson torsion
+        seismograph.
+
+    """
+
+    if obspy_def:
+        # Create Wood-Anderson response - ObsPy values
+        woodanderson = {"poles": [-6.283185 - 4.712j,
+                                  -6.283185 + 4.712j],
+                        "zeros": [0j],
+                        "sensitivity": 2080,
+                        "gain": 1.}
+    else:
+        # Create Wood Anderson response - different to the ObsPy values
+        # http://www.iris.washington.edu/pipermail/sac-help/2013-March/001430.html
+        woodanderson = {"poles": [-5.49779 + 5.60886j,
+                                  -5.49779 - 5.60886j],
+                        "zeros": [0j],
+                        "sensitivity": 2080,
+                        "gain": 1.}
+
+    if convert in ('DIS2DIS', 'VEL2VEL'):
+        # Add an extra zero to go from disp to disp or vel to vel.
+        woodanderson['zeros'].extend([0j])
+
+    return woodanderson
+
+
+class Archive:
+    """
+    The Archive class handles the reading of archived waveform data. It is
+    capable of handling any regular archive structure. Requests to read
+    waveform data are served up as a `QMigrate.data.SignalData` object. Data
+    will be checked for availability within the requested time period, and
+    optionally resampled to meet a unified sampling rate. The raw data read
+    from the archive will also be retained.
```

Comment:
```suggestion
    The Archive class handles the reading of archived waveform data.
    
    It is capable of handling any regular archive structure. Requests to read
    waveform data are served up as a `QMigrate.data.SignalData` object. Data
    will be checked for availability within the requested time period, and
    optionally resampled to meet a unified sampling rate. The raw data read
    from the archive will also be retained.
```

Reply:
This needs to be applied to the updated commit with `SignalData` replaced with `WaveformData`. But I agree with the re-formatting.